allennlp/semparse/contexts/table_question_context.py

```diff
@@ -232,21 +336,21 @@ def get_entities_from_question(self) -> Tuple[List[Tuple[str, str]], List[Tuple[
             normalized_token_text = self.normalize_string(token_text)
             if not normalized_token_text:
                 continue
-            token_column = self._string_in_table(normalized_token_text)
-            if token_column:
-                token_type = token_column.split(":")[0].replace("_column", "")
+            token_columns = self._string_in_table(normalized_token_text)
+            if token_columns:
+                token_type = token_columns[0].split(":")[0].replace("_column", "")
```

Comment:
You had a comment explaining the `[0]` in `_string_in_table`, but you removed it when moving this logic here.  I'd  move the comment here too.